static/js/pong.js

```diff
@@ -0,0 +1,127 @@
+(function() {
```

Comment:
Did you create this game?

Reply:
Yes, out of boredom on a flight a few weeks ago.

Reply:
Hah, looks nice! :) Feel free to include your attribution in it if you want to. Also, I tried playing it, and there might be a small bug. The bats can go out of the screen, so maybe set a boundary while you're at it?